modules/SofaConstraint/MappingGeometricStiffnessForceField.inl

```diff
@@ -52,6 +52,14 @@ void MappingGeometricStiffnessForceField<DataTypes>::addKToMatrix(const sofa::co
 
     const sofa::defaulttype::BaseMatrix* mappingK = l_mapping->getK();
 
+
+
+    if (mappingK==NULL)
+    {
+        msg_warning("can not get the stiffness matrix of the mapping ");
```

Comment:
1) Is there any solution or hints that could guide the users on how they should fix their scene ?